packages/manager/src/components/CircleProgress/CircleProgress.tsx

```diff
@@ -75,54 +14,108 @@ interface Props extends CircularProgressProps {
   const value = typeof props.value === 'number' ? props.value : 0;
 
-  return mini ? (
-    <CircularProgress
-      className={classNames({
-        [classes.mini]: true,
-        [classes.noPadding]: noPadding,
-        [classes.tag]: tag,
-      })}
-      size={noPadding ? 22 : 40}
-      aria-label="Content is loading"
-      data-qa-circle-progress
-      data-testid="circle-progress"
-      tabIndex={0}
-    />
-  ) : (
-    <div
-      className={`${className} ${classes.root}`}
-      aria-label="Content is loading"
-    >
+  if (mini) {
+    return (
+      <StyledMiniCircularProgress
+        size={noPadding ? 22 : 40}
+        aria-label="Content is loading"
+        data-qa-circle-progress
+        data-testid="circle-progress"
+        tabIndex={0}
+      />
+    );
+  }
+
+  return (
+    <StyledRootDiv className={className} aria-label="Content is loading">
       {children !== undefined && (
-        <div className={classes.valueInside}>{children}</div>
+        <Box sx={{ position: 'absolute', marginTop: 4 }}>{children}</Box>
       )}
       {noInner !== true && (
-        <div className={classes.topWrapper}>
-          <div className={classes.top} />
-        </div>
+        <StyledTopWrapperDiv>
+          <StyledTopDiv />
+        </StyledTopWrapperDiv>
       )}
-      <CircularProgress
+      <StyledCircularProgress
         {...rest}
-        className={classes.progress}
         size={124}
         value={value}
         variant={variant}
         thickness={2}
         data-qa-circle-progress={value}
         data-testid="circle-progress"
       />
-    </div>
+    </StyledRootDiv>
   );
 };
 
-export default CircleProgressComponent;
+export { CircleProgress };
+
+const StyledRootDiv = styled('div')(({ theme }) => ({
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'center',
+  margin: '0 auto 20px',
+  position: 'relative',
+  width: '100%',
+  [theme.breakpoints.up('md')]: {
+    flex: 1,
+    height: 300,
+  },
+}));
+
+const StyledTopWrapperDiv = styled('div')(({}) => ({
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'center',
+  position: 'absolute',
+  width: '100%',
+  height: '100%',
+}));
+
+const StyledTopDiv = styled('div')(({ theme }) => ({
+  width: 70,
+  height: 70,
+  borderRadius: '50%',
+  border: '1px solid #999',
+  [theme.breakpoints.up('sm')]: {
+    width: 120,
+    height: 120,
+  },
+}));
+
+const StyledCircularProgress = styled(CircularProgress)<CircleProgressProps>(
+  ({ theme }) => ({
+    position: 'relative',
+    [theme.breakpoints.down('sm')]: {
+      width: '72px !important',
+      height: '72px !important',
+    },
+  })
+);
+
+const StyledMiniCircularProgress = styled(
+  CircularProgress
+)<CircleProgressProps>(({ theme, ...props }) => ({
+  padding: `calc(${theme.spacing()} * 1.3)`,
+  ...(props.noPadding && {
+    padding: 0,
+  }),
+  ...(props.tag && {
+    width: '12px !important',
+    height: '12px !important',
```

Comment:
Does using `&&` here rather than `!important` work? I thought it seemed to, but might have missed something because I also tried to see if we could avoid `!important` for the `StyledCircularProgress` at the `sm` breakpoint and I wasn't seeing that work.
```suggestion
'&&': {
    width: '12px',
    height: '12px',
    }
```

Reply:
@mjac0bs It looks like this style wasn't even being used in https://github.com/linode/manager/blob/develop/packages/manager/src/components/Tag/Tag.tsx#L150-L161 so I went ahead and removed it. Can you check again?

Reply:
I think you're right. Your changes LGTM.